client/src/components/CreateProduct.js

```diff
@@ -113,6 +115,8 @@ import {
                     <TextField 
                         id = "qty" 
                         className= {classes.root} 
+                        aria-label="quantity-input"
```

Comment:
Aria labels are intended to be 'human readable' strings: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/listbox_role

Reply:
I fixed the string values for the aria-labels